tools/breaking-change-detector/README.md

```diff
@@ -0,0 +1,39 @@
+# Breaking change detector
+
+## Purpose
+Detects breaking changes between provider versions.
+
+Specifically protects customer expectations between [minor version](https://www.terraform.io/plugin/sdkv2/best-practices/versioning#example-minor-number-increments).
+
+
+## Execution of
+
+### Program:mode-default
+```bash
+go run . -providerVersion="google"
```

Comment:
How does someone provide the necessary versions to run against? Does this assume the user has already set up the go mod replace correctly? How would they figure out which things to modify

Reply:
I'll have to tackle these new workflows. You bring up good points.